src/Mvc/Mvc.Core/src/MvcCoreLoggerExtensions.cs

```diff
@@ -1017,12 +1017,12 @@ public static void LocalRedirectResultExecuting(this ILogger logger, string dest
             _localRedirectResultExecuting(logger, destination, null);
         }
 
-        public static void ObjectResultExecuting(this ILogger logger, object value)
+        public static void ObjectResultExecuting(this ILogger logger, ObjectResult result, object value)
```

Comment:
minor but: now that we're passing in the result, we don't need to pass in value as a separate parameter, we can get it from `result.Value`.

Reply:
I have come across the following issue which I think makes this suggestion invalid. 
https://github.com/dotnet/aspnetcore/issues/11607

Please confirm this suggestion is no longer needed.